src/clr-angular/popover/signpost/signpost.spec.ts

```diff
@@ -97,6 +103,50 @@ export default function(): void {
         expect(this.ifOpenService.open).toBe(false);
       });
     });
+
+    describe('aria-control values', () => {
+      spec(ClrSignpost, TestDefaultSignpost, ClrSignpostModule);
+
+      function checkAriaControlsId(id: string, element: HTMLElement) {
+        const triggerControlsValue = element.querySelector('.signpost-action').getAttribute('aria-controls');
+        const closeControlsValue = element
+          .querySelector('.signpost-content .signpost-action')
+          .getAttribute('aria-controls');
+        const contentId = element.querySelector('.signpost-content').getAttribute('id');
+
+        expect(id).toBe(contentId);
```

Comment:
This is great but is it clear where this breaks when it breaks?

If not, we may want to add the optional string on the expect...

`expect(id).toBe(whatWeAreTesting, 'string that is passed that describes what we are testing and why it might be broken')`

Reply:
I will add the string to the expect for more descriptive failures.